spec/collectionspace/mapper/data_mapper_core_spec.rb

```diff
@@ -162,7 +162,8 @@
         end
 
         it 'maps as expected' do
-          @fixture_xpaths.each do |xpath| puts xpath
```

Comment:
Remove `puts` or `puts xpath` from the end of this.

If you wanted me to see the xpaths for troubleshooting, `puts xpath` could be moved to the next line, but it's Ruby style to end the opening of a block (`@fixture_xpaths.each do |xpath|`) with the block arguments (between `|`). Any thing that happens within the block should be on a subsequent line.

Reply:
Noted.